crypto/kzg/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "kzg"
+version = "0.1.0"
+authors = ["Pawan Dhananjay <pawandhananjay@gmail.com>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ethereum_ssz = "0.5.0"
+ethereum_ssz_derive = "0.5.3"
+tree_hash = "0.5.2"
+derivative = "2.1.1"
+serde = "1.0.116"
+serde_derive = "1.0.116"
+ethereum_serde_utils = "0.5.0"
+hex = "0.4.2"
+ethereum_hashing = "1.0.0-beta.2"
+c-kzg = { git = "https://github.com/ethereum/c-kzg-4844", rev = "fa3c62989527073fdce8b2138bb27a52bb2407c5" , features = ["mainnet-spec"]}
+c_kzg_min = { package = "c-kzg", git = "https://github.com/ethereum//c-kzg-4844", rev = "fa3c62989527073fdce8b2138bb27a52bb2407c5", features = ["minimal-spec"], optional = true }
+arbitrary = { version = "1.0", features = ["derive"], optional = true }
```

The arbitrary feature isn't really optional. Everything fails to compile without it. This is fine though, it's what most of our crates do now.

```suggestion
arbitrary = { version = "1.0", features = ["derive"] }
```